copy_old_es_database.py

```diff
@@ -6,9 +6,10 @@
 from requests_aws4auth import AWS4Auth
 
 OLD_AWS_PROFILE_NAME = 'default'
+NEW_AWS_PROFILE_NAME = 'cool-dns-amazonesfullaccess'
```

Comment:
Do we need to include this profile in our set of profiles used with [`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync)?

Reply:
No, it's a temporary one I created to give me temporary permissions to do whatever I wanted to the DB.  Normally access is granted via Cognito and a role associated with that.  That is the bit that I hand-jammed before, but I have a note to revisit and code up correctly in the COOL.

Reply:
That's what I figured, but just wanted to confirm. 👍